inc/blocks/block-widget-categories.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Categories Block Widget
+ *
+ * @package Bootscore
+ * @version 5.4.0
+ */
+
+
+// Exit if accessed directly
+defined('ABSPATH') || exit;
+
+
+/**
+ * Categories Block
+ */
+if (!function_exists('bootscore_block_widget_categories_classes')) {
+  /**
+   * Adds Bootstrap classes to categories block widget.
+   *
+   * @param string $block_content The block content.
+   * @param array  $block         The full block, including name and attributes.
+   * @return string The filtered block content.
+   */
+  function bootscore_block_widget_categories_classes($block_content, $block) {
+    
+    // Check if the block contains the 'wp-block-categories-list' class, exclude the dropdown.
+    if (strpos($block_content, 'wp-block-categories-list') !== false) {
+      $search  = array(
+        'wp-block-categories-list',
+        'cat-item',
+        'current-cat',
+        '<a',
+        '(',
+        ')'
+      );
+      $replace = array(
+        'wp-block-categories-list bs-list-group list-group',
+        'cat-item list-group-item list-group-item-action d-flex justify-content-between align-items-center',
+        'current-cat active',
+        '<a class="stretched-link text-decoration-none"',
+        '<span class="badge bg-primary-subtle text-primary-emphasis">',
+        '</span>'
+      );
+
+      $block_content = str_replace($search, $replace, $block_content);
+    }
+
```

Comment:
Here we can also add our own filters potentially, so people can change the content after our changes (could be done in #575). Either that or they can hook after the `render_block_core/categories` hook. What do you say?

Reply:
Yes, good idea! 

Think those filters are a huge step and I can't wait for https://github.com/bootscore/bootscore/pull/575. BTW, we should add a filter for those `py-5 mt-5` classes as well:

```php
<div id="content" class="site-content <?= apply_filters('bootscore_container_class', 'container', 'page'); ?> py-5 mt-5">
```

Think this is very useful if users removes the `fixed-top` in `header.php`.

Reply:
My idea is to merge this PR, and work on adding more filters (also for these changes) in #575. Agree?

Reply:
Full agree👍